apps/launch/components/Path/steps/Design.tsx

```diff
@@ -0,0 +1,38 @@
+import { FileDoneOutlined } from '@ant-design/icons';
+
+import { UNICODE_SYMBOLS } from 'libs/util-constants/src';
+
+import { StepContent, StepList } from '../StepContent';
+import { StepComponentProps } from '../types';
+
+const putTogetherData = {
+  title:
+    'Put together a technical document that specifies how each agent in your economy will be designed, architected and how they interact with one another.',
+  items: [
+    'What do you want the individual agents to do?',
+    'How many agent instances of each type are required?',
+    <>
+      Translate KPI definitions in a staking contract specification. See the{' '}
+      <a
+        href="https://github.com/valory-xyz/autonolas-aip/blob/aip-2/docs/OlasAutomate.pdf"
```

Comment:
```suggestion
        href="https://docs.google.com/document/d/1U2H8VJK019WfxdQpFY2u5TjSEDxlzXma7vrWpYeip90"
```

Reply:
@oaksprout the access is restricted from the outside, is that expected? this may prevent users from get the info quickly

Reply:
Ah – let me check with Ralph. I suggest we comment out this sentence for now.

Reply:
done